src/Periphery/Executor.sol

```diff
@@ -8,6 +8,7 @@ import { LibAsset } from "../Libraries/LibAsset.sol";
 import { ILiFi } from "../Interfaces/ILiFi.sol";
 import { IERC20Proxy } from "../Interfaces/IERC20Proxy.sol";
 import { TransferrableOwnership } from "../Helpers/TransferrableOwnership.sol";
+import { IERC721Receiver } from "@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol";
```

Comment:
Is there any reason to actually import this?

Reply:
for explicity purpose and the comments in the file I would tend to include it. But I forgot to let the Executor inherit from it, which I changed now.